blockchain/blockchain-wallet-v4-frontend
# Fix/tb sb widget bugs

## Description (optional)
Remove unneeded US flag. Correct Bugs with amount field causing overflow.

## Testing Steps (optional)
US Flag should no longer be shown. to test the amount:

1. Open new tab
2. Go to http://localhost:8080/#/open/simple-buy?crypto=BTC&amount=25000000000
3. Amount should be truncated with ellipses
